imap/imap.go

```diff
@@ -0,0 +1,346 @@
+package imap
+
+// Functionality taken from https://github.com/jprobinson/eazye
+
+import (
+	"bytes"
+	"crypto/tls"
+	"fmt"
+	"strconv"
+	"time"
+
+	log "github.com/gophish/gophish/logger"
+	"github.com/gophish/gophish/models"
+	"github.com/jordan-wright/email"
+	"github.com/mxk/go-imap/imap"
+
+	_ "github.com/paulrosania/go-charset/data"
+)
+
+// IMAPEmail Email struct with IMAP UID
```

Comment:
I think throughout the `imap` package, there's room to clean up what we call things. For example, here we can just call this `Email`. This makes it easier to use outside of the package since it wouldn't be as repetitive (`imap.Email` vs. `imap.IMAPEmail`).

The same goes for things like:

* `ValidateIMAP` -> `Validate`
* `newIMAPClient` -> `newClient`
* `monitorIMAP` -> `monitor`

Reply:
Good point. go-lint had told me off for this. Done.